lightning/src/ln/channel.rs

```diff
@@ -6607,11 +6679,6 @@ impl<SP: Deref> InboundV1Channel<SP> where SP::Target: SignerProvider {
 			counterparty_initial_commitment_tx.to_broadcaster_value_sat(),
```

Comment:
Because we moved the block that decrements `cur_counterparty_commitment_transaction_number`, this needs to be changed to:

```
self.context.cur_counterparty_commitment_transaction_number + 1,
```